bar-trender-front-end/src/components/ApiLoginForm.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+class POSTLoginForm extends React.Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      login: {
+        email: "client1@email.com",
+        password: "holamundo."
+      },
+      input: {},
+      errors: {}
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleLogin = this.handleLogin.bind(this);
+  }
+
+  handleLogin() {
+    alert('A form was submitted: ' + JSON.stringify(this.state));
+
+    // this.props.searchEngine(this.state.term);
+    /* fetch('https://127.0.0.1:8000/authentication/login/', {
+         method: 'POST',
+        // We convert the React state to JSON and send it as the POST body
+        body: JSON.stringify(this.state.login)
+       }).then(function(response) {
+         console.log(response)
+         console.log("SE HA LOGUEADO DE LOCOS")
+         return response.json();
+       });*/
+
+  }
+
+  handleChange(event) {
+    let input = this.state.input;
+    input[event.target.name] = event.target.value;
+    this.setState({
+      input
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    if (this.validate()) {
+      console.log(this.state);
+      let input = {};
+      input["email"] = "";
+      input["password"] = "";
+      this.setState({ input: input });
+      const response = fetch("http://127.0.0.1/authentication/login",
+        {
+          method: 'POST',
+          body: JSON.stringify(this.state.login),
+          headers: {
+            'X-Api-Key': 'apikeytest',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
                "X-Api-Key": 'apikeytest',
```